### ISSUE TITLE
json from Aalto is still using the old version

### ISSUE THREAD

**Issue by popabczhang** [CityScope/CS_CityIO_Backend#34]
@agrignard mentioned: 
"Speaking about Aalto how come they can produce a json that is still the old school version. I think that raise an issue on the GH side (see https://github.com/CityScope/CS_Rhino_Grasshopper_Template/issues/1). We have to be careful that any new comer is at least using our new template otherwise we loose the track"

**Comment by popabczhang** [comment]
@agrignard, the json composer and sender was updated to be compliant with the new json format at commit [`493614d`](https://github.com/CityScope/CS_Rhino_Grasshopper_Template/commit/493614d70a2ddbb308966a4c14abe043d3c9a5cb). I emailed the Aalto researchers for the update. 
![image](https://user-images.githubusercontent.com/2801912/36827237-4f360de0-1ce0-11e8-92d0-311603ac2cff.png)
I will send a reminder email again for them to update to the latest version.
